MODULE.bazel

```diff
@@ -4,15 +4,18 @@ module(
     compatibility_level = 1,
 )
 
-bazel_dep(name = "bazel_features", version = "1.19.0")
+bazel_dep(name = "bazel_features", version = "1.28.0")
 bazel_dep(name = "bazel_skylib", version = "1.7.1")
 bazel_dep(name = "platforms", version = "0.0.10")
 bazel_dep(name = "protobuf", version = "27.0", repo_name = "com_google_protobuf")
 
 cc_configure = use_extension("//cc:extensions.bzl", "cc_configure_extension")
 use_repo(cc_configure, "local_config_cc", "local_config_cc_toolchains")
 
-register_toolchains("@local_config_cc_toolchains//:all")
+register_toolchains(
+    "@local_config_cc_toolchains//:all",
```

Comment:
Was this previously set somewhere else? How was the CC toolchain registered without this?

Reply:
No, Bazel didn't have a test toolchain registered anywhere. That's okay though, `cc_test` only has an optional requirement so far, but for our purposes it has to be turned into a mandatory one. At that point I will also be able to delete the legacy fallback logic.